src/css/profile/wearable/changeable/theme-circle/arclistview.less

```diff
@@ -78,7 +78,39 @@
 				margin-top: 0;
 			}
 		}
-		
+
+		&.ui-li-swipe-left {
+			.ui-swipe-icon {
+				position: absolute;
+				width: 98 * @px_base;
+				height: 100%;
+				right: 0;
+				clip-path: inset(0 0 0 100%);
+
+				&-background {
+					position: absolute;
+					width: 100%;
+					height: 100%;
+					background: #FFFFFF;
+					border-top-right-radius: 54 * @px_base;
+					border-bottom-right-radius: 54 * @px_base;
+					opacity: 0.15;
+				}
+
+				// Due to opacity, this can not be child of background.
+				// https://css-tricks.com/non-transparent-elements-inside-transparent-elements/
+				&-delete {
+					position: absolute;
+					content: '';
+					width: 76 * @px_base;
+					height: 76 * @px_base;
+					left: 50%;
+					top: 50%;
+					transform: translate(-50%, -50%);
+					background: rgba(255, 0, 255, 0.5) url('images/tw_list_delete_ic.png');
```

Comment:
In my opinion background color is not part of framework this is only for guideline needs

Reply:
Replaced this ui-swipte-icon-background with ui-swipte-icon::after in less only. Now, example looks like:
`                        <div class="ui-swipe-icon">`
    `                            <div class="ui-swipe-icon-delete"></div>`
                        `</div>`
Thanks.

Reply:
I have on mind this pink background color. I think it shouldn't be part of framework.

Reply:
Ok, removed.